src/js/steps/letters/generateHtml.js

```diff
@@ -134,9 +138,9 @@ module.exports = function (options) {
         // removing all current content within the container, adding the
         // letters element to the DOM (and saving it to the data object), and
         // adding the loading gif in whilst the book loads.
-        $book.empty();
-        data.lettersElement = $lettersContainer.appendTo($book);
-        data.loading = data.loading.appendTo($book);
+        // $book.empty();
+        data.lettersElement = $lettersContainer.prependTo($book);
```

Comment:
It's to do with the ordering that things had to by initialised when I was fixing the bug with the container being emptied to early.  It's possible it could be append and that was left over from a fix, but it works ¯\_(ツ)_/¯